copy_monitoring_plugin.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+#For when you've built the monitoring plugin locally and want this in your next stack
+cp ../openfire-monitoring-plugin/target/monitoring.jar ./xmpp/clustered/1/plugins/monitoring.jar
+cp ../openfire-monitoring-plugin/target/monitoring.jar ./xmpp/clustered/2/plugins/monitoring.jar
```

Comment:
Unless you've customized the plugin building, the generated plugin file name will be `monitoring-openfire-plugin-assembly-jar` - Is that something that should be used here?

Reply:
Good spot. Yep. That's my custom build name. 🤦‍♂️

Reply:
Fixed.